jaeger-crossdock/README.md

```diff
@@ -0,0 +1,16 @@
+# Dockerized permutations testing for Jaeger
+
+This repository implments docker testing for jaeger-client-java.  To be more specific Jaeger 
+uses a frame work called [Crossdock](https://github.com/crossdock/crossdock) for testing
+propagation between Jaeger clients in different languages.
+
+# Usage #
+
+## Run Crossdock tests ##
+
+In order to run crossdock tests running `make crossdock-fresh` in the root directory of this
+project is sufficient.
+
+# Contributors #
+
+To push a new crossdock image run `make upload-image`.
```

Comment:
"Contributors" section is customary used to list contributors to the project, not instructions for contributors (CONTRIBUTING.md is used for the latter).

I don't like this new make command to begin with, and we certainly shouldn't put it in the readme. Uploading images to dockerhub is done by the CI builds, there's no reason to do that manually.